cmake/MMDeploy.cmake

```diff
@@ -12,6 +12,18 @@ function (mmdeploy_export NAME)
             RUNTIME DESTINATION bin)
 endfunction ()
 
+macro(mmdeploy_add_net NAME)
+    if (MMDEPLOY_DYNAMIC_BACKEND)
+        # remove unused
+        cmake_parse_arguments(_NET "EXCLUDE;STATIC;SHARED;MODULE" "" "" ${ARGN})
```

Comment:
I don't think it's necessary to parse the argument, because the callings are:
```
mmdeploy_add_net(${PROJECT_NAME} trt_net.cpp)
mmdeploy_add_net(${PROJECT_NAME} torch_net.cpp)
...
```
There are no 'option' or 'one keyword arg' or 'multi keywords args'